src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -6302,8 +6302,8 @@ void FillRemainingPredefinedEntries(EnergyPlusData &state)
                 totalArea += thisZone.FloorArea * zoneMult;
 
                 // minimum dynamic target ventilation Voz-dyn-min
-                PreDefTableEntry(state, state.dataOutRptPredefined->pdchOaMvMinDynTrgVent, thisZone.Name, thisZonePreDefRep.VozMin, 3);
-                totalVozMin += thisZonePreDefRep.VozMin * zoneMult;
+                PreDefTableEntry(state, state.dataOutRptPredefined->pdchOaMvMinDynTrgVent, thisZone.Name, thisZonePreDefRep.VozMin / zoneMult, 3);
+                totalVozMin += thisZonePreDefRep.VozMin;
```

Comment:
Here the incoming value is already multiplied, so divide by `zoneMult` for the table entry and don't multiply for the facility total. Similar changes for others in this source file and in SizingManager.